easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1_fix-alphafold-tests.patch

```diff
@@ -0,0 +1,22 @@
+Fix `run_alphafold_test.py` to use an unique directory per test to avoid pollution when the tests are run multiple times
+Patch by Simon Branford (University of Birmingham)
```

Comment:
@branfosj Let's refer to the upstream issue too here: https://github.com/deepmind/alphafold/issues/365

Reply:
done in e17e3b4